Header header
uint16[1024] distance_data
float32[1024] x
float32[1024] y
float32[1024] z
uint16[1024] confidence 
uint16[1024] amplitude
float32[4] amplitude_normalization
uint32 available
uint16 height
uint16 width

float32 trans_x
float32 trans_y
float32 trans_z
float32 rot_x
float32 rot_y
float32 rot_z

